packages/versions/src/bin.ts

```diff
@@ -1,3 +1,7 @@
 import { runVersions } from './cli';
 
-runVersions();
+runVersions()
+  .then(() => process.exit(0))
+  .catch(() => {
+    process.exit(1);
+  });
```

Comment:
Technically, this could be considered a breaking change.

But only for someone extending our CLIs.

Reply:
Annoyingly I think we should exercise caution and just be hard on every breaking change, but I like the use case, this comes up a lot.

I'd opt for a `fuels version:check` command that implements this, and maybe we add to the deprecation issue that `fuels version:check` becomes `fuels:version`.

Reply:
Left a suggestion for an alternative: https://github.com/FuelLabs/fuels-ts/pull/3108#issuecomment-2346355732